### ISSUE TITLE
Visualize.py returns me "TypeError: Empty 'DataFrame': no numeric data to plot" at 57th line.

### ISSUE THREAD

**Issue by Lumenified** [TulsaBioinformaticsToolsmith/LtrDetector#10]
I use windows and installed MSYS2 to compile with gcc. used 10.2.0. After I did some modifications, everything works flawlessly. However I get an error when I try to visualize them. Im sure that I do it with a correct way but it doesn't accept `sys.arg[3]`. I also installed the requirements so I think its about py file :/ 
[GCA_003401745.1_ASM340174v1_genomicDetector.bed.txt](https://github.com/BioinformaticsToolsmith/LtrDetector/files/5024845/GCA_003401745.1_ASM340174v1_genomicDetector.bed.txt)
[GCA_003401745.1_ASM340174v1_genomicRawScores.csv.txt](https://github.com/BioinformaticsToolsmith/LtrDetector/files/5024847/GCA_003401745.1_ASM340174v1_genomicRawScores.csv.txt)
Here are the files.

**Comment by Lumenified** [comment]
I believe that my file format is wrong, I thought .fna file format would work. Sorry, my bad. Everything works flawlessly 👍

**Comment by Lumenified** [comment]
I tried with the assembly file for Drosophila melanogaster, but I got another error when I tried to visualize it.
[dm6Detector.bed.txt](https://github.com/BioinformaticsToolsmith/LtrDetector/files/5035596/dm6Detector.bed.txt)
[dm6RawScores.csv.txt](https://github.com/BioinformaticsToolsmith/LtrDetector/files/5035599/dm6RawScores.csv.txt)
Here are the files. I think I have a problem in my analysis 😐 I couldn't be sure. I think I should exclude the chromosome X and Y and should keep the files separately like each chromosome in different file and in a folder. Is it correct ?
